test/e2e/set-manifest-flags.ts

```diff
@@ -29,6 +29,7 @@ function parseIntOrUndefined(value: string | undefined): number | undefined {
 export async function setManifestFlags(flags: ManifestFlags = {}) {
   if (process.env.CI) {
     flags.ci = {
+      ...flags.ci,
```

Comment:
What gets missed if this is not included?

Reply:
it is now reverted in dbbb9ba6bd667f6a86ecb64399b6eaca7f6f54c1